dir.props

```diff
@@ -155,6 +155,7 @@
     <!-- Central place to set the versions of all nuget packages produced in the repo -->
     <PackageVersion Condition="'$(PackageVersion)' == ''">1.2.0</PackageVersion>
     <WindowsAPISetPackageVersion>1.0.1</WindowsAPISetPackageVersion>
+    <SystemNativePackageVersion>4.4.0-beta-24721-02</SystemNativePackageVersion>
     
```

Comment:
Fortunately this can be auto-updated based on the full package version, not just that prerelease string: I suggest moving this property to [dependencies.props](https://github.com/dotnet/coreclr/blob/a60661ea96f6760d6cfd8e1e86c370f7e7052ed0/dependencies.props#L13-L17) and adding a new [`XmlUpdateStep`](https://github.com/dotnet/coreclr/blob/a60661ea96f6760d6cfd8e1e86c370f7e7052ed0/dependencies.props#L45-L49) for that property.